libraries/AP_Param/AP_Param.cpp

```
@@ -78,6 +78,8 @@ void AP_Param::eeprom_write_check(const void *ptr, uint16_t ofs, uint8_t size)
     _storage.write_block(ofs, ptr, size);
 }
 
+bool AP_Param::_hide_disabled_groups = false;
```

Is this correct? By default we don't hide disabled groups?